src/sentry/web/forms/accounts.py

```diff
@@ -288,11 +288,6 @@ def clean_user(self):
                     "The account you are trying to recover is managed and does not support password recovery."
                 )
             )
-
-        if len(users) > 1:
-            raise forms.ValidationError(
-                _("Multiple accounts were found matching this email address.")
-            )
```

Comment:
This error was another place we would leak information. Now the 'first' user will get a password reset email.

Reply:
Hmmmmmm. I wonder if this is a scenario that’s even possible today. This might be legacy. Wanna investigate?

Reply:
There are way more than 0. After talking with Matt we're going to treat this case like a missing user and not give any feedback, and not email the potentially wrong person as well.